### DISCUSSION TITLE
FaceID/TouchID Security

### DISCUSSION THREAD

**Discussion by freedreamer82** [RavenSystem/esp-homekit-devices#2193] [HAA Home App]
Should be possible to add faceId/touchId  to login the app? 
Since from it u have the chance to update devices should be interesting to increase security.

Thanks

**Comment by xrust83** [comment]
it makes no sense, since you can enter setup mode in at least 4 ways except for haa-manager

What is the point of putting a high-tech gate with an expensive technological lock and a golden key if there are 3 holes in the fence on the side of the gate 😂

**Reply by freedreamer82** [reply]
no sense for you , at least. I would find more safe to protect the APP with faceID , I u don't like u should  disable  the feature.

**Reply by xrust83** [reply]
> no sense for you , at least. I would find more safe to protect the APP with faceID , I u don't like u should  disable  the feature.
> 
> 

https://youtu.be/na7wxLqsZtE?si=pF-OpoeNbog7kRdd

**Reply by freedreamer82** [reply]
Thanks , it works but should be Better integrated in the app cause with this method you come back to lockscreen every time. I hope he will integrate in the app btw
